### ISSUE TITLE
Can't create automated build

### ISSUE THREAD

**Issue by carlitux** [docker/hub-feedback#401]
on dev tools csv_repo_name: "This field is required." and default values are not being set I have to change values if not I am having errors.

**Comment by rrarunan** [comment]
@carlitux can you let me know if it is a bitbucket or a github repo.. and if its a public repo share the actual link, so I can take a look? Thanks!

**Comment by carlitux** [comment]
It is a public github
https://github.com/linkux-it/strider-docker-slave-node

**Comment by rrarunan** [comment]
@carlitux I'm trying to repro this, and cannot. Can you please share a screenshot of what you are entering in the form so, we can debug further? And also, the screenshot of the 400/error message you see in the devtools will be great. Thank you!

**Comment by carlitux** [comment]
First is 400 is about the cvs_repo_name as I put previously... But first
there are others if I don't select my username... All data is the default
filled... If you need any other please let me know and I will do when I am
on the computer.

**Comment by carlitux** [comment]
I followed on chrome from list of organization and worked but I refresh the page doesn't

**Comment by rrarunan** [comment]
Ok, the first case where you follow it from the list of organization is what is supported. If you can create a build that way, I would recommend doing it. 

But, to understand your other case: refreshing the page, which page are you trying to refresh and create the build? The form itself?

**Comment by carlitux** [comment]
Well I followed from organizations (github)... to test (hub docket) I created two organizations on hub but on form is selected my username, repo-name, master branch.  

1.- Click on create (don't show errors but server respond with 403, I noticed this after refresh)
2.- Here is when I refreshed
3.- Open dev tools (Here I noticed the 403)
4.- Change to one organization (hub) then again to my username
5.- Click create (Shows error on description, asking that is required)
6.- Returns 400 with the csv_repo_name (didn't show error messages)

After this I did all from organizations without refresh on step 6 the automated build was created.

Just to suggest: I noticed you are using react and I am assuming somethings...
1.- on refresh one csv_repo_name state/prop is missed maybe collect from url (router) could solve
2.- select owner/context be sure the selected one on browser is set on prop/state

Thoso both are the ones I had problems.
